homebrew_challenge/src/pages/Overview.jsx

```diff
@@ -204,6 +205,16 @@ const Overview = ({
           </Col>
         </Row>
         <Row>
+          <Col>
+            <hr className="full-width-hr d-none d-sm-flex" />
```

Comment:
Here the  d-none d-sm-flex is in the hr, but its in the Row components elsewhere. Should perhaps be on line 207 instead for consistency

Reply:
moved